src/kernel/arch/x86/paging.zig

```diff
@@ -0,0 +1,122 @@
+const std = @import("std");
```

Comment:
Could u also add unit tests to functions that are not state related like map_dir_entry, map_dir, alignBackward...

Reply:
Certainly